src/locales/en.json

```diff
@@ -13,7 +13,7 @@
       "label": "Precautions",
       "environment_header": "Environment",
       "environment_question": "Is it indoor or outdoor? Is the space ventilated, or is the air filtered?",
-      "no_intimate_bonus_outdoors": "Due to very close distances, we are not confident that being outdoors reduces the risk in a substantial way. Thus, we are not providing any bonus for being outdoors when intimate.",
+      "no_close_bonus_environment": "Good environment choices are being ignored in the calculation. We are not confident that good ventilation or being outdoors strongly reduces the risk when people are this close to each other.",
```

Comment:
Kind of awkward wording?

"Good things are being ignored"

Also, changing the json key will cause all existing translations to show the English until they are re-translated. I think it is preferable to show the current translations (keep "no_intimate_bonus_outdoors")